src/mono/System.Private.CoreLib/src/System/Threading/ThreadPool.Browser.Mono.cs

```diff
@@ -110,12 +110,13 @@ internal static bool NotifyWorkItemComplete(object? threadLocalCompletionCountOb
             throw new PlatformNotSupportedException();
         }
 
-        [DynamicDependency("Callback")]
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        private static extern void QueueCallback();
+        internal static extern unsafe void MainThreadScheduleBackgroundJob(void* callback);
```

Comment:
nit: It would be nice to collect all of these internal calls that are called inside S.P.C into a single class